src/pages/index.js

```diff
@@ -20,88 +87,176 @@ const cardSection = new Section(
   ".gallery__cards"
 );
 
-// Render initial cards
-cardSection.renderItems();
+function fillCardLike(){
+  const likeBtn = document.querySelector('.card__like-button');
+  likeBtn.classList.toggle('card__like-button--active');
```

Comment:
Please, delete it

Now only the `Card` class knows every **card selector**, so you need to encapsulate the likes logics inside its methods and call them in `index.js`. It’s a violation of **Single responsibility** principle